pkg/plugins/policies/meshcircuitbreaker/plugin/v1alpha1/plugin.go

```diff
@@ -90,17 +91,14 @@ func applyToInbounds(
 			Port:    iface.DataplanePort,
 		}
 
-		rules, ok := fromRules.Rules[listenerKey]
-		if !ok {
-			continue
-		}
-
 		cluster, ok := inboundClusters[envoy_names.GetLocalClusterName(iface.DataplanePort)]
 		if !ok {
 			continue
 		}
 
-		if err := configure(rules, subsetutils.MeshElement(), cluster); err != nil {
+		conf := inbound2.MatchesAllIncomingTraffic[api.Conf](fromRules.InboundRules[listenerKey])
```

Comment:
import names with a number almost always look bad, please use `rules_inbound`

Reply:
right, I forgot to update it